### ISSUE TITLE
cdktf azurerm: Maven repository metadata out of date

### ISSUE THREAD

**Issue by Zman94** [hashicorp/terraform-cdk#3010]
### Expected Behavior

https://repo.maven.apache.org/maven2/com/hashicorp/cdktf-provider-azurerm/maven-metadata.xml should display the actual latest cdktf azurerm version as well as all versions

### Actual Behavior

https://repo.maven.apache.org/maven2/com/hashicorp/cdktf-provider-azurerm/maven-metadata.xml shows latest version is
```
<latest>0.4.24</latest>
<release>0.4.24</release>
```

### Steps to Reproduce

1. Set up your environment
```
❯ cdktf debug
cdktf debug
language: typescript
cdktf-cli: 0.17.1
node: v18.16.0
cdktf: 0.17.1
constructs: 10.2.69
jsii: null
terraform: 1.4.6
arch: x64
os: linux 5.15.0-1041-azure
```
In package.json
```
    "jsii": "~5.1.7",
    "jsii-pacmak": "~1.84.0",
```
2. `npx jsii`
3. `npx jsii-pacmak -v`
4. See the following debug lines
```
#STDOUT> [INFO] Downloading from $workspaces$terraform-cdk-modules$dist$java: file:///workspaces/terraform-cdk-modules/dist/java/com/hashicorp/cdktf-provider-azurerm/maven-metadata.xml
#STDOUT> [INFO] Downloaded from central: https://repo.maven.apache.org/maven2/com/hashicorp/cdktf-provider-azurerm/maven-metadata.xml (10 kB at 1.0 MB/s)
```

### Versions

❯ cdktf debug
cdktf debug
language: typescript
cdktf-cli: 0.17.1
node: v18.16.0
cdktf: 0.17.1
constructs: 10.2.69
jsii: null
terraform: 1.4.6
arch: x64
os: linux 5.15.0-1041-azure

### Providers

❯ cdktf provider list
───────────┬────────────┬──────┬───────────────────────────┬────────────┐
───         ─────        ──     ──       ──────             ────
 Provider  │ Provider   │ CDKTF│          Package Name     │ Package    │
Name        Version             Constrain                   Version 
───────────┼────────────┼──────┼───────────────────────────┼────────────┤
───         ─────        ──     ──       ──────             ────
 carlpett/s│ 0.7.2      │      │ 0.7.2                     │            │
ps
───────────┼────────────┼──────┼───────────────────────────┼────────────┤
───         ─────        ──     ──       ──────             ────
 azurerm   │ 3.63.0     │      │          @cdktf/provider-a│ 9.0.2      │
                         ^0.17.0         urerm

───────────┴────────────┴──────┴───────────────────────────┴────────────┘
───         ─────        ──     ──       ──────             ────

### Possible Solutions

It looks like a package script is failing to update the latest version

### Workarounds

N/A. Possibly I could manually download from the correct version, though I'm not familiar enough with Maven to make that work.
https://repo.maven.apache.org/maven2/com/hashicorp/cdktf-provider-azurerm/9.0.2/

### Anything Else?

https://discuss.hashicorp.com/t/unable-to-package-with-maven-and-jsii/56078/2

### Help Wanted

- [ ] I'm interested in contributing a fix myself

### Community Note

- Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
- Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
- If you are interested in working on this issue or have submitted a pull request, please leave a comment

**Comment by jsteinich** [comment]
Related: https://github.com/hashicorp/terraform-cdk/issues/2996

**Comment by Zman94** [comment]
Closing this ticket. Root cause was identified and solved in #2996. Thanks all!
